bin/node/runtime/src/impls.rs

```diff
@@ -142,49 +114,84 @@ mod tests {
 	}
 
 	#[test]
-	fn fee_multiplier_update_poc_works() {
-		let fm = Multiplier::saturating_from_rational(0, 1);
+	fn truth_value_update_poc_works() {
+		let fm = Multiplier::saturating_from_rational(1, 2);
 		let test_set = vec![
 			(0, fm.clone()),
 			(100, fm.clone()),
+			(1000, fm.clone()),
 			(target(), fm.clone()),
 			(max() / 2, fm.clone()),
 			(max(), fm.clone()),
 		];
 		test_set.into_iter().for_each(|(w, fm)| {
 			run_with_system_weight(w, || {
 				assert_eq_error_rate!(
-					fee_multiplier_update(w, fm),
-					TargetedFeeAdjustment::<TargetBlockFullness>::convert(fm),
-					// Error is only 1 in 10^18
-					Multiplier::from_inner(1),
+					truth_value_update(w, fm),
+					runtime_multiplier_update(fm),
+					// Error is only 1 in 100^18
+					Multiplier::from_inner(100),
 				);
 			})
 		})
 	}
 
 	#[test]
-	fn empty_chain_simulation() {
-		// just a few txs per_block.
-		let block_weight = 0;
-		run_with_system_weight(block_weight, || {
-			let mut fm = Multiplier::default();
+	fn multiplier_can_grow_from_zero() {
+		// if the min is too small, then this will not change, and we are doomed forever.
+		// the weight is 1/10th bigger than target.
```

Comment:
```suggestion
		// the weight is 1/100th bigger than target.
```

this is a very important test!